src/main/java/de/geolykt/enchantments_plus/CustomEnchantment.java

```diff
@@ -888,33 +894,33 @@ public void setEnchantment(ItemStack stk, CustomEnchantment ench, int level, Wor
                 meta.getPersistentDataContainer().set(ench.key, PersistentDataType.SHORT, (short) level);
                 lore.add(ench.getShown(level, world));
             }
-        
+
             //Disenchant item
             if (ench != null &&
                     level <= 0 &&
                     meta.getPersistentDataContainer().has(ench.key, PersistentDataType.SHORT)) {
                 meta.getPersistentDataContainer().remove(ench.key);
-            }
-            
+                    }
```

Comment:
Also a malindention here
```suggestion
                }
```